symengine/cwrapper.cpp

```diff
@@ -170,34 +189,46 @@ TypeID basic_get_type(const basic s)
     return static_cast<TypeID>(s->m->get_type_code());
 }
 
-void symbol_set(basic s, const char *c)
+int symbol_set(basic s, const char *c)
 {
+    CWRAPPER_BEGIN();
     s->m = SymEngine::symbol(std::string(c));
+    CWRAPPER_END(...,-1);
```

Comment:
Since all the wrappers use `CWRAPPER_END(...,-1)`, why not to make it the default, and just call `CWRAPPER_END()`.

Regarding the empty `()`, I think you can remove those too from both the definition and the call.